### PR TITLE
Day6

### PR DESCRIPTION
**Pull request by Pckvr**

### CODE REVIEW

File: src/main/scala/Day6/HashMapAnagrams.scala [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package Day6
+
+import scala.io.Source
+
+class HashMapAnagrams {
+
+def storeLinesInArray(filename: String) = {
+  val source = Source.fromFile(filename)
+  val linesArray = source.getLines.toArray
+  linesArray
```
Comment (tvaidotas) [inline src/main/scala/Day6/HashMapAnagrams.scala]: the whole method body could be this one line statement `Source.fromFile(filename).getLines.toArray` although this is not a safe way to read from file as you're not handling any errors, what if the path you give is broken, or the file isn't there

File: src/main/scala/Day6/HashMapAnagrams.scala [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+  def extractWordFromLine(filename: String) = {
+    val source = Source.fromFile(filename)
+    val wordsArray = source.getLines().toString()
+    println(wordsArray)
```
Comment (tvaidotas) [inline src/main/scala/Day6/HashMapAnagrams.scala]: same issue here

File: src/main/scala/Garage/MongoConnection.scala [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import scala.util.{Failure, Success}
+
+class MongoConnection {
+
+  val mongoClientPath = "//localhost"
+  val databaseName = "garage"
+
+  def getConnection(): MongoClient = {
+    val mongoClient = MongoClient("mongodb:" + mongoClientPath)
+    mongoClient
```
Comment (tvaidotas) [inline src/main/scala/Garage/MongoConnection.scala]: why do you assign the value to variable and immediately return it, just return it in the first place

File: src/main/scala/Garage/MongoConnection.scala [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  }
+
+  def closeCollection() = {
+    getConnection().close()
+  }
+
+  def getDatabase(): MongoDatabase = {
+    val database = getConnection().getDatabase(databaseName)
```
Comment (tvaidotas) [inline src/main/scala/Garage/MongoConnection.scala]: why do you assign the value to variable and immediately return it, just return it in the first place

File: src/main/scala/Garage/MongoConnection.scala [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    database
+  }
+
+  def getCollection(collectionName: String): MongoCollection[Document] = {
+    val collection = getDatabase().getCollection(collectionName)
```
Comment (tvaidotas) [inline src/main/scala/Garage/MongoConnection.scala]: why do you assign the value to variable and immediately return it, just return it in the first place

File: src/main/scala/Day6/HashMapAnagrams.scala (hunk 1 above)
Comment (Pckvr) [reply src/main/scala/Day6/HashMapAnagrams.scala]: I am going to start this exercise again

**Pull request merged**